test/e2e/operator_test.go

```diff
@@ -351,14 +351,14 @@ func TestMutation(t *testing.T) {
 
 			testNamespace := &corev1.Namespace{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: "e2e-" + strings.ToLower(t.Name()),
+					Name: "e2e-" + strings.ToLower(test.name),
```

Comment:
test.name contains spaces which are not valid characters in the namespace name

Reply:
Creating a whole operator PR takes less commits than this PR :). Once job fails, I'm going to fix this